lib/ansible/modules/user.py

```diff
@@ -3244,6 +3264,42 @@ def modify_user(self):
             if rc is not None and rc != 0:
                 self.module.fail_json(name=self.name, msg=err, rc=rc)
 
+        # Manage user settings
+        uid = user_info[2]
+        if self.uid is not None:
+            uid = self.uid
+
+        passwd_entry = [
+            self.name,
+            'x',
+            to_native(uid),
+            to_native(gid),
+            self.comment or user_info[4],
+            self.home or user_info[5],
+            self.shell or user_info[6],
+        ]
+
+        contents = []
+        change = False
+        with open(self.PASSWORDFILE, 'r') as password_file:
+            for line in password_file:
+                if line.startswith(self.name):
+                    fields = line.strip().split(':')
+                    if fields != passwd_entry:
+                        change = True
+                        line = ':'.join(passwd_entry) + '\n'
+
+                contents.append(line)
+
+        if change:
+            rc = 0
+            if not self.module.check_mode:
+                tmpfd, tmpfile = tempfile.mkstemp(dir=self.module.tmpdir)
+                with open(tmpfile, 'w') as f:
+                    f.writelines(contents)
+
+                self.module.atomic_move(tmpfile, self.PASSWORDFILE)
```

Comment:
I would call `backup_local` right before this point.

Reply:
I had that in there originally and was wondering if I should add that back in. `backup` isn't in the module parameters, so there's no way to turn it off. I'm not sure how folks would feel about that, but I suppose it's better safe than sorry.